### PR TITLE
fix all fixable broken macros in markdown

### PR DESCRIPTION
**Pull request by fiji-flo**
#### Summary
This a result of starting to enable flaw fixing for markdown.

#### Supporting details
These changes get applied already when building content. I wanted to feed that back into the source.

#### Metadata

- [ ] Adds a new document
- [x] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/mozilla/firefox/releases/10/index.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ Great progress has been made to update IndexedDB to the latest draft specificati
...
 - Two new values for the CSS property {{ cssxref("unicode-bidi") }} have been added: `-moz-isolation` and `-moz-plaintext`. The `-moz-isolation` value isolates, from a directionality point of view, the element from its environment, letting it have a different directionality. An element with `unicode-bidi:-moz-isolation` behaves like a {{ HTMLElement("bdi") }} element. The `-moz-plaintext` indicates the browser to use the Unicode browser heuristic to determine directionality and not the CSS {{ cssxref("direction") }} property.
-- The CSS {{ cssxref("linear-gradient()") }} and {{ cssxref("repeating-linear-gradient()") }} properties have been updated to support the new `to` syntax and the _magic corner_ algorithm. This allows to give a precise color on the corner of a gradient-filled box.
+- The CSS {{ cssxref("gradient/linear-gradient") }} and {{ cssxref("gradient/repeating-linear-gradient") }} properties have been updated to support the new `to` syntax and the _magic corner_ algorithm. This allows to give a precise color on the corner of a gradient-filled box.
```
Comment (teoli2003) [inline files/en-us/mozilla/firefox/releases/10/index.md:99]: This should be like this:
```suggestion
- The CSS {{ cssxref("gradient/linear-gradient", "linear-gradient()") }} and {{ cssxref("gradient/repeating-linear-gradient", "repeating-linear-gradient()") }} properties have been updated to support the new `to` syntax and the _magic corner_ algorithm. This allows to give a precise color on the corner of a gradient-filled box.
```

File: files/en-us/mozilla/firefox/releases/11/index.md [hunk 2]
```diff
@@ -72,7 +72,7 @@ _No change._
 - Previous to Firefox 11, object store {{domxref("IDBObjectStore.autoIncrement","autoIncrement")}} counters were shared across all object stores for a given database, whereas per spec each object store should have a separate counter. This is now fixed.
 - It is now possible to {{domxref("IDBObjectStore.createIndex","create an index")}} with an empty `keyPath`.
 - It is now possible to create a multi-entry index (see [`IDBObjectStore.createIndex` parameters](/en-US/docs/Web/API/IDBObjectStore/createIndex#parameters).)
-- The {{event("abort")}} event now bubbles; in addition, an {{domxref("IDBDatabase.onabort")}} handler has been added.
+- The {{event("abort")}} event now bubbles; in addition, an {{domxref("IDBTransaction.abort_event")}} handler has been added.
```
Comment (teoli2003) [inline files/en-us/mozilla/firefox/releases/11/index.md:75]: We never let `xxyy.event` being displayed.
```suggestion
- The {{event("abort")}} event now bubbles; in addition, an {{domxref("IDBTransaction.abort_event", "onabort")}} handler has been added.
```

File: files/en-us/mozilla/firefox/releases/24/index.md [hunk 3]
```diff
@@ -31,7 +31,7 @@ tags:
 - Support for the {{domxref("Comment.Comment", "Comment()")}} constructor has been added ({{bug(869006)}}).
 - Support for the {{domxref("DocumentFragment.DocumentFragment", "DocumentFragment()")}} constructor has been added ({{bug(869002)}}).
 - The {{domxref("FocusEvent")}} interface has been implemented ({{bug(855741)}}).
-- Support for the {{domxref("ChildNode.remove()")}} method has been added ({{bug(856629)}}).
+- Support for the {{domxref("Element.remove")}} method has been added ({{bug(856629)}}).
```
Comment (teoli2003) [inline files/en-us/mozilla/firefox/releases/24/index.md:34]: This is a method -> we write with parenthesis at the end.
```suggestion
- Support for the {{domxref("Element.remove()")}} method has been added ({{bug(856629)}}).
```

**Review by teoli2003 — COMMENTED** [review]
Having all of these fixable broken macros in one PR is useful, but we need to review and edit per our conventions. 

I can do this for this PR if you want, when ready.

**Comment by fiji-flo** [comment]
I realize this PR need more care then the broken-links. I'll land the changes to yari which I used to generate this. And we tackle broken macros in smaller chunks. Closing for now.

**Pull request closed without merging**